### PR TITLE
Fix theming guide link

### LINKED ISSUE #2461: 404 - link is broken for the theme guide page
Go to: https://github.com/angular/material2/blob/master/guides/getting-started.md

Then click on that page this link: 

https://github.com/angular/material2/blob/master/guides/guides/theming.md

404 in your face!

### PR DESCRIPTION
**Pull request by gyscos**
Fixes #2461

### CODE REVIEW

**Comment by gyscos** [comment]
I signed it!

File: guides/getting-started.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ export class PizzaPartyAppModule { }
 This is **required** to apply all of the core and theme styles to your application. You can either
 use a pre-built theme, or define your own custom theme.
 
-:trident:  See the [theming guide](guides/theming.md) for instructions.
+:trident:  See the [theming guide](./theming.md) for instructions.
 
 ### Additional setup for gestures
 Some components ()`md-slide-toggle`, `md-slider`, `mdTooltip`) rely on
```
Comment (DevVersion) [inline guides/getting-started.md]: Just noticed that we have a small typo here. Can you remove the `)` parenthesis while you're at it?

**Comment by DevVersion** [comment]
@jelbourn We should get this in 😄 We likely got **7** PR's for the same change

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**